src/PackagePublisher/PackagePublisher.nuspec

```diff
@@ -8,5 +8,6 @@
   </metadata>
   <files>
     <file src="netcoreapp1.0\" target="/" />
+    <file src="netcoreapp1.0\build\*.targets" target="/build" />
```

Comment:
Nothing. Can be removed. Can't remember why I added it even though I wrote this last night